packages/core/admin/admin/src/components/AutoReloadOverlayBlocker/Portal.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect } from 'react';
+import ReactDOM from 'react-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import PropTypes from 'prop-types';
+import Content from './Content';
+import Overlay from './Overlay';
+import Wrapper from './Wrapper';
+
+const overlayContainer = document.createElement('div');
+const ID = 'autoReloadOverlayBlocker';
+overlayContainer.setAttribute('id', ID);
+
+const Portal = ({ className, displayedIcon, description, title, elapsed, isOpen }) => {
```

Comment:
WYT of renaming this one? Portal is a generic term and it looks this component is very specific no?

Reply:
What do you have in mind?